app/views/admin/products/show.html.haml

```diff
@@ -5,27 +5,6 @@
 		%h1.text-xl.font-semibold.mb-8.overflow-hidden.text-ellipsis.whitespace-nowrap
 			= "(#{@product.brand} - #{@product.weight})"
 
-	.flex.flex-col.justify-between.items-center{ class: "md:w-2/5 sm:w-full mb-8" }
-		.flex.flex-row
-			.flex.flex-col{ class: "w-1/2 px-4" }
-				%label.mb-4.max-w-xs.overflow-hidden.text-ellipsis.whitespace-nowrap
-					= "Number of servings"
-				%input{type: "number", name: "servings", id: "servings", required: true, class: "rounded-lg p-2 w-full border-0 bg-slate-50 focus:ring-primary-500", min: "0", value: "1", disabled: true}
-			.flex.flex-col{ class: "w-1/2 px-4" }
-				%label#listbox-label
-				= "Serving size"
-				.relative.mt-4
-					%button#weight-units-list-button.relative.w-full.rounded-lg.pl-3.pr-10.text-left.bg-slate-50.shadow-sm{"aria-expanded" => "true", "aria-haspopup" => "listbox", "aria-labelledby" => "listbox-label", class: "py-2 focus:outline-none focus:ring-1 focus:ring-primary-500 sm:text-sm sm:leading-6", type: "button", disabled: true}
```

Comment:
Make sure to remove the corresponding JavaScript implementation for the `weight-units`, `weight-units-list-button` and the remaining classes that dealt with the display of the serving input field and the weight unit dropdown menu.